result/bgpprefix.yaml

```diff
@@ -169,6 +169,17 @@ components:
         communities:
           x-include: '#/components/schemas/BgpPrefix.State/properties/communities'
           x-field-uid: 7
+        extended_communities:
+          description: >-
+            Optional received Extended Community attributes.
+            Each received Extended Community attribute is available for retrieval in two forms.
+            Support of the 'raw' format in which all 8 bytes (16 hex characters) is always present and available for use.
+            In addition, if supported by the implementation, the Extended Community attribute may also be retrieved in the 
+            'structured' format which is an optional field.
+          type: array
+          items:
+            $ref: '#/components/schemas/BgpPrefix.RecvExtendedCommunity'          
```

Comment:
is Recv prefix is required as get state is all about learned/received routes related only.

Reply:
Changed all prefixes to Result.x and removed Recv Prefix for this.